app/marketing/views.py

```diff
@@ -624,7 +627,8 @@ def leaderboard(request, key=''):
     else:
         amount_max = 0
 
-    is_linked_to_profile = '_tokens' in key or '_keywords' in key
+    profile_keys = ['_tokens', '_keywords', '_cities', '_countries', '_continents']
+    is_linked_to_profile = any(sub in key for sub in profile_keys)
```

Comment:
Thanks for updating these with the original PR feedback! 💯

Reply:
Would love to take credit, but think @owocki made this change (not sure why GitHub is showing otherwise....maybe a squash commit?)